membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -216,6 +216,7 @@ object AccountDetails {
     case _: Product.Paper => "subscriptions" // Paper includes GW 🤦‍
     case _: Product.ZDigipack => "subscriptions"
     case _: Product.SupporterPlus => "recurringSupport"
+    case _: Product.TierThree => "recurringSupport"
```

Comment:
there was a question about this (for S+) and how that affects things in manage, @rBangay  woud know more about the implications.